react/features/mobile/callkit/middleware.js

```diff
@@ -235,10 +235,13 @@ function _conferenceWillJoin({ getState }, next, action) {
     const state = getState();
     const url = getInviteURL(state);
     const hasVideo = !isVideoMutedByAudioOnly(state);
+    const { callUUID } = state['features/base/config'];
 
     // When assigning the call UUID, do so in upper case, since iOS will
     // return it upper cased.
-    conference.callUUID = uuid.v4().toUpperCase();
+    conference.callUUID = callUUID
+        ? callUUID.toUpperCase() : uuid.v4().toUpperCase();
```

Comment:
You can write it more concisely:

`conference.callUUID = (callUUID || uuid.v4()).toUpperCase();`